modules/querying/pages/select-statement/FROM Clause (v2)

```diff
@@ -0,0 +1,372 @@
+= FROM Clause (v2)
```

Comment:
```suggestion
= FROM Clause (Syntax V2)
:navtitle: From Clause (V2)

```
@Tushar-TG-14 Note a couple of details:
1. In the body of the document, V1 and V2 are capitalized, so it should be capitalized in the title also. 
2. I decided it's better to have "Syntax V2" in the title to be clear we are talking about GSQL Syntax, not about the database version.
3. BUT, I still want the navigation title to be short. There is a special directive for setting the navigation title differently than the page title.
4. There should be a blank line between the title and the body.